pass-an-image.cpp

```diff
@@ -8,5 +8,11 @@ int main() {
 	std::cout << "Brightening a 512 x 512 image\n";
     ImageBrightener brightener(image);
     int attenuatedCount = brightener.BrightenWholeImage();
+    if (image.pixels != NULL)
+    {
+        image.pixels[0] = '\0';
+        // code...
+        free(image.pixels);
```

Comment:
Good use of `free` to release allocated memory :)
But this means all users of the `Image` struct will need to do this. Why not do this within that struct itself?